.release/vault-k8s-artifacts.hcl

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: BUSL-1.1
```

Comment:
I think this repo is [MPL](https://github.com/hashicorp/vault-k8s/blob/main/LICENSE)? 

```suggestion
# Copyright (c) HashiCorp, Inc.
# SPDX-License-Identifier: MPL-2.0
```